app/views/posts/new.html.erb

```diff
@@ -13,3 +16,7 @@
     <%= form.submit %>
   </p>
 <% end %>
+<% end %>
+<% if not user_signed_in? %>
```

Comment:
`else` > `if not`

Reply:
I tried using `else` but it would error? Specifically I tried `<% if ...%> <% else %> <% end %>` which is what someone else did online